.github/workflows/test.yml

```diff
@@ -172,10 +172,10 @@ jobs:
           toolchain: stable
           override: true
           components: rustfmt, clippy
-      - name: Install Python 3.6
+      - name: Install Python 3.7
         uses: actions/setup-python@v1
         with:
-          python-version: "3.6"
+          python-version: "3.7"
       - name: Test python
         run: make python-test
```

Comment:
I'm not sure we should bump this version — we generally run tests against the minimum version of $lang that we support, regardless of whether that version is EOL.

Reply:
updated back to `3.6`